app/views/admin/health/patient_referrals/_actions.haml

```diff
@@ -10,8 +10,14 @@
         %button.btn.btn-secondary{type: 'button', data: {toggle: 'dropdown'}, aria: {haspopup: true, expanded: false}}
           Actions
           %span.caret
-        .dropdown-menu.dropdown-menu-right{style: 'padding:10px;'}
-          %div{style: 'width: 300px;'}
+        .dropdown-menu.dropdown-menu-right{style: 'padding:10px; white-space: nowrap;'}
+          %div{style: 'padding: 0.25rem 1.5rem;'}
```

Comment:
This doesn't look like a new thing exactly, but I wonder if some of our helper classes could clean this up a bit.
Totally untested:
```suggestion
        .dropdown-menu.dropdown-menu-right.p-2.nobr
          .pl-2.pr-2.pb-4.pt-4
```